packages/@adobe/spectrum-css-temp/vars/spectrum-large.css

```diff
@@ -244,6 +244,8 @@
   --spectrum-meter-large-border-radius: 4px;
   --spectrum-meter-small-border-radius: 3px;
   --spectrum-pagination-page-button-line-height: 32px;
+  --spectrum-overlay-positive-transform-distance: 8px;
```

Comment:
We should add these directly in the file, not as global values
They should also be derived from other values if possible

Reply:
this should be derived from `--spectrum-global-dimension-size-75`, but I dont know how to turn this global value negative without using `calc()` 😞 

Edge does not support using translate, calc and var together... thats why new var that is equal to calculation result is added... and vars are different depending on size scale

Reply:
Yep, I ran into this issue as well with Switch and did the same kind of fix https://github.com/adobe-private/react-spectrum-v3/commit/b2bb7eea825f4e6fb0164efd46a7fcfd0cbe5db2. Looks like spectrum-css is waiting for chromium edge to roll out for this issue as well: https://github.com/adobe/spectrum-css/issues/427

Maybe we keep this hack in for now but just include a comment somewhere so we can remove when chromium edge becomes the defacto browser for windows?